EthOn.rdf

```diff
@@ -374,6 +373,21 @@
 
 
 
+    <!-- http://ethon.consensys.net/hasAuthorBeneficiary -->
+
+    <owl:ObjectProperty rdf:about="http://ethon.consensys.net/hasAuthorBeneficiary">
+        <rdfs:subPropertyOf rdf:resource="http://ethon.consensys.net/hasBeneficiary"/>
+        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#FunctionalProperty"/>
+        <rdfs:domain rdf:resource="http://ethon.consensys.net/Block"/>
+        <rdfs:range rdf:resource="http://ethon.consensys.net/Account"/>
+        <suggestedStringRepresentation>hasAuthorBeneficiary</suggestedStringRepresentation>
+        <rdfs:comment>Relates a Block to an Account to which the fees from gas costs, the static mining reward, and the reward for including uncles are transferred.</rdfs:comment>
```

Comment:
There is a comma before "and"
`static mining reward, and the reward for including`